info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 2    | 2.2.1    | HEADER_MISSING_OR_WRONG_COPYRIGHT         | Checks: copyright exists on top of the file and is properly formatted (as a block comment)<br>Fix: adds copyright if it is missing and required                                                                                                                  | yes | isCopyrightMandatory, copyrightText (sets the copyright pattern for your project, you also can use `;@currYear;` the key word in your text in aim to indicate the current year, instead of explicitly specifying)                    |                                                                                                                                                                                                                               |
+| 2    | 2.2.1    | WRONG_COPYRIGHT_YEAR                      | Checks: copyright has a valid year<br>Fix: makes the year valid                                                                                                                                                                                               | yes | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
+| 2    | 2.2.1    | HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE   | Check: warns if a file with zero or >1 classes doesn't have a KDoc header                                                                                                                                                                                       | no  | no                                                                                                                                                                                                                         |                                                                                                                                                                                                                               |
```

Comment:
warns if a file with zero or >1 classes
What does it mean? 
zero or >1 means any number of classes. Explanations needed.